care/security/authorization/template.py

```
@@ -0,0 +1,61 @@
+    def can_write_facility_template(self, user, facility):
+        """
+        Check if the user has permission to write templates in the facility
+        """
+        return self.check_permission_in_facility_organization(
+            [TemplatePermissions.can_write_template.name],
+            user,
+            facility=facility,
+            root=True,
+        )
+
+    def can_preview_template(self, user, facility):
```

Ruff (0.14.6)

Unused method argument: `facility`

(ARG002)